client/app/scripts/contrast-compiler.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable class-methods-use-this */
+// Webpack plugin for creating contrast mode stylesheet
+const _ = require('lodash');
+
+function findAsset(collection, name) {
+  return _.find(collection, c => _.includes(c, name));
+}
+
+module.exports = class ContrastStyleCompiler {
+  apply(compiler) {
+    let themeJsChunk;
+
+    compiler.plugin('compilation', (compilation) => {
+      compilation.plugin('html-webpack-plugin-before-html-processing', (htmlPluginData, callback) => {
+        themeJsChunk = findAsset(htmlPluginData.assets.js, 'contrast-theme');
+        if (!themeJsChunk) {
+          return callback(null, htmlPluginData);
+        }
+        // Find the name of the contrast stylesheet and save it to a window variable.
+        const { css, publicPath } = htmlPluginData.assets;
+        const contrast = findAsset(css, 'contrast-theme');
+        const normal = findAsset(css, 'style-app');
+        // Convert to JSON string so they can be parsed into a window variable
+        const themes = JSON.stringify({ normal, contrast, publicPath });
+        // Append a script to the end of <head /> to evaluate before the other scripts are loaded.
+        const script = `<script>window.__WEAVE_SCOPE_THEMES = JSON.parse('${themes}')</script>`;
+        const [head, end] = htmlPluginData.html.split('</head>');
+        htmlPluginData.html = head.concat(script).concat(end);
```

Comment:
When I run `npm run build` locally, I get the following `index.html` file:

```
<!doctype html>
<html class="no-js"><head><link href="style-app-4134b6394afe6c264f1e.css?c044bfbe96884908980f" rel="stylesheet"></head>
  <head>
    <meta charset="utf-8">
    <title>Weave Scope</title>
    <meta name="description" content="">
    <meta name="viewport" content="width=device-width, initial-scale=1">
  <script>window.__WEAVE_SCOPE_THEMES = JSON.parse('{"normal":"style-app-4134b6394afe6c264f1e.css?c044bfbe96884908980f","contrast":"style-contrast-theme-fd61e9572aa66a409ca1.css?c044bfbe96884908980f","publicPath":""}')</script>
  <body>
    <!--[if lt IE 10]>
      <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
    <![endif]-->
    <div class="wrap">
      <div id="app"></div>
    </div>
  <script type="text/javascript" src="vendors-098b95ea8f29c00c4a2f.js?c044bfbe96884908980f"></script><script type="text/javascript" src="app-4134b6394afe6c264f1e.js?c044bfbe96884908980f"></script></body>
</html>
```

So it actually contains two `<head>` tags, with the second one missing a closure `</head>`. Funny enough, it works just fine on my Chrome, but I still think it should be fixed before going to prod :)